.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: "release"
+on:
+  push:
+    tags:
```

Comment:
Can we use a branch? Users of some plugin managers prefer branches https://github.com/nvim-tree/nvim-tree.lua/issues/1944

Reply:
It should be possible to trigger the build only for certain branches.
But at the moment, the GitHub action uses tags to generate the release versions.

I do plan to see if it's possible to use GitHub releases, too: https://github.com/nvim-neorocks/luarocks-tag-release/issues/8

Reply:
> But at the moment, the GitHub action uses tags to generate the release versions.

Good enough for me. tags _are_ the correct approach for versions.

If there is user pushback they can investigate options.